### PR TITLE
DOCS | Update fb app creation steps

### PR DESCRIPTION
**Pull request by immber**
## What does this PR do?
Updates plugin documentation for talk-plugin-facebook-auth

## How do I test this PR?
Verify netlify preview

### CODE REVIEW

**Review by kgardnr — APPROVED** [review]

**Review by alinhle — CHANGES_REQUESTED** [review]
As in our sync, @immber is going to update the instructions to the new version of _Facebook for Developers_ where they changed the language and the locations of the settings.

File: plugins/talk-plugin-facebook-auth/README.md [hunk 1]
```diff
@@ -27,24 +27,23 @@ You can learn more about getting a Facebook App ID at the
...
+* Add a "Product" (Under "Products" click + to add a Product):
+    * Setup "Facebook Login"
     * choose `www` 
     * enter your Talk domain url
     * click _Next_ several times to get through the add code steps (You do not need to modify any code, the plugin takes care of this part for you.)
-* Go back to Settings > Basic:
-    * enter the callback url (Use your Talk domain with this endpoint: `/api/v1/auth/facebook/callback`)
+* Under Product Settings:
+    * set Valid OAuth Redirect URIs to your callback url (Use your Talk domain with this endpoint: `/api/v1/auth/facebook/callback`)
     * Locate your App Id and App Secret, set these as config vars on your instance of Talk
```
Comment (alinhle) [inline plugins/talk-plugin-facebook-auth/README.md:47]: Maybe add a line here: You can find both under `Settings -> Basic`

File: plugins/talk-plugin-facebook-auth/README.md [hunk 2]
```diff
@@ -27,24 +27,23 @@ You can learn more about getting a Facebook App ID at the
...
 * Toggle the "Live" button to make app live
```
Comment (alinhle) [inline plugins/talk-plugin-facebook-auth/README.md]: Maybe add: "Toggle the "Live" button at the top bar to make app live.

**Review by alinhle — APPROVED** [review]
Thanks Jessie! I added some suggestions. Feel free to incorporate them or not. You can merge when you are done :-)

File: plugins/talk-plugin-facebook-auth/README.md (hunk 1 above)
Comment (immber) [reply plugins/talk-plugin-facebook-auth/README.md:47]: This is the one that wasn't actually under Settings > Basic, it really is under the product settings, so was incorrect before, but is correct now.

File: plugins/talk-plugin-facebook-auth/README.md (hunk 2 above)
Comment (immber) [reply plugins/talk-plugin-facebook-auth/README.md]: Making this change!

**Pull request merged**